staging/src/k8s.io/apiserver/pkg/admission/cel/metrics_test.go

```diff
@@ -59,7 +59,28 @@ func TestNoUtils(t *testing.T) {
             apiserver_validating_admission_policy_check_total{enforcement_action="allow",params="params.example.com",policy="policy.example.com",policy_binding="binding.example.com",state="active",validation_expression="true"} 1
 			`,
 			observer: func() {
-				Metrics.ObserveCheck(context.TODO(), time.Duration(10)*time.Second, "policy.example.com", "binding.example.com", "true", "allow", "params.example.com", "active")
+				Metrics.ObserveAdmission(context.TODO(), time.Duration(10)*time.Second, "policy.example.com", "binding.example.com", "true", "params.example.com", "active")
```

Comment:
This metric currently is not for admission but for admission with error right? So maybe either change the name here or change from controller to make it for all admission?

Reply:
Fixed - I've renamed it to `ObserveAdmissionWithError`